src/jekyll/jk_errors.rs

```diff
@@ -0,0 +1,43 @@
+#![cfg_attr(feature = "cargo-clippy", allow(redundant_closure))]
```

Comment:
How come this is needed?

Reply:
Just blindly copy pasted from `error.rs`